remove top left
  Sort by: Name - 3/9

must be in UTF-8 format!

--- a/addons/skin.estuary/xml/Includes.xml	2018-08-07 11:31:56.172671553 +0200
+++ b/addons/skin.estuary/xml/Includes.xml	2018-08-07 11:33:11.365769065 +0200
@@ -763,7 +763,7 @@
 		</control>
 	</include>
 	<include name="TopBar">
-		<param name="sublabel">$INFO[Container.SortMethod,$LOCALIZE[31022]: ,  ∙  ]$INFO[Container.CurrentItem,, / ]$INFO[Container.NumItems]</param>
+		<param name="sublabel">$INFO[Container.CurrentItem,, / ]$INFO[Container.NumItems]</param>
 		<definition>
 			<control type="group">
 				<animation effect="slide" end="0,-112" time="300" tween="sine" easing="inout" condition="$EXP[infodialog_active]">conditional</animation>
